baseapp/baseapp.go

```diff
@@ -938,12 +972,7 @@ func (app *BaseApp) DefaultPrepareProposal() sdk.PrepareProposalHandler {
 func (app *BaseApp) DefaultProcessProposal() sdk.ProcessProposalHandler {
```

Comment:
a bit out of scope but should we remove these as methods and have them as plugins. it would show people how to set it up

Reply:
Plugins? Do you have an example? You mean removing them on `BaseApp` and defining them as standalone?

Reply:
yea as standalone, thanks for fixing my nomenclature

Reply:
Gotcha. Pushed. LMK what you think.